frontend/src/pages/Tomorrow.jsx

```diff
@@ -27,29 +26,26 @@ function Tomorrow() {
           <img src={Pic1} alt="vegetables" />
         </div>
         <div className="paragraph">
-          <h2>Manger Local et de Saison</h2>
+          <h2>Eating Local and Seasonal</h2>
           <p>
-            L'une des premières étapes pour penser à demain dans votre cuisine
-            est de privilégier les produits locaux et de saison. En optant pour
-            des aliments qui sont cultivés à proximité de chez vous et qui sont
-            récoltés au moment opportun, vous réduisez l'empreinte carbone
-            associée au transport des denrées alimentaires sur de longues
-            distances. Les légumes de saison sont non seulement plus frais et
-            savoureux, mais ils contribuent également à soutenir les
-            agriculteurs locaux.
+            One of the first steps to thinking about tomorrow in your kitchen is
+            to favor local and seasonal products. By opting for foods that are
+            grown near you and are harvested at the right time, you reduce the
+            carbon footprint associated with the transport of foodstuffs over
+            long distances. Seasonal vegetables are not only fresher and tasty,
+            but they also help support local farmers.
           </p>
         </div>
       </div>
 
       <div className="text-container">
         <div className="paragraph">
-          <h2>Réduire le Gaspillage Alimentaire</h2>
+          <h2>Reduce Food Waste</h2>
           <p>
-            Le gaspillage alimentaire est un problème majeur dans le monde
-            d'aujourd'hui. En réduisant le gaspillage dans votre cuisine, vous
-            contribuez à la préservation des ressources naturelles. Une astuce
-            simple consiste à planifier vos repas, à conserver les restes et à
-            être créatif avec les ingrédients que vous avez sous la main.
+            Food waste is a major problem worldwide of today. By reducing waste
+            in your kitchen, you contribute to the preservation of natural
+            resources. A tip simple is to plan your meals, save leftovers and be
+            creative with the ingredients you have on hand.
```

Comment:
Cadeau => Food waste is a significant issue in today's world. By minimizing waste in your kitchen, you play a part in conserving natural resources. A simple tip is to plan your meals, store leftovers, and get creative with the ingredients you have on hand.